.circleci/config.yml

```diff
@@ -0,0 +1,104 @@
+orbs:
+  slack: circleci/slack@3.4.1
+version: 2.1
+jobs:
+  test:
+    machine:
+      image: ubuntu-1604:201903-01
+    steps:
+      - checkout
+      - run: docker run --name postgres -p 5432:5432 -d postgres:10.6
```

Comment:
no need

Reply:
ah yes, you don't have postgres here, makes sense

Reply:
and now the tests are failing because there's no redis, and it tries to connect not to local redis, but to `redis-gsuite-wrapper`, which is not there. I'll fix it in another PR, if you don't mind, as well as eslint and yamllint